docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+}
+``` 
+
+Sample response:
+```json
+{
+  "_index": "detect_text_test",
+  "_id": "1",
+  "_version": 1,
+  "result": "created",
+  "_shards": {
+    "total": 2,
+    "successful": 1,
+    "failed": 0
+  },
+  "_seq_no": 0,
+  "_primary_term": 1
+}
+
+``` 
+Let's get the indexed document and check if the extracted texts are showing under the `text` field`:
+
+```json
+GET detect_text_test/_doc/1
+``` 
+Sample response:
+
+```json
+{
+    "_index": "detect_text_test",
+    "_id": "1",
+    "_version": 1,
+    "_seq_no": 0,
+    "_primary_term": 1,
+    "found": true,
+    "_source": {
+        "text": [
+            "Platform",
+            "Search for anything",
+            "Platform",
+            "Search",
+            "for",
+            "anything"
```

Comment:
These four results are words detected. They are duplicate with the first two results which are detected lines. Can we just return detected lines ?

Reply:
I was looking at the the documentation example and it looks like the API returns lines and words lines represent how we read it (search for anything) and the words are individual part (search, for, anything). 

- https://docs.aws.amazon.com/rekognition/latest/dg/text-detecting-text-procedure.html

I asked chat if it were possible to filter (using only LINE) on the fly not sure if this works but worth a try `$..TextDetections[?(@.Type=='LINE')].DetectedText`

Reply:
Rekognition will detect the word or line of text recognized. Even though the words seems duplicated, the individual words can be used for tokenized word in search, they can be handy.

We can prefer keeping it this way, and suggest users to refer to rekognition API if they would like to do further filtering.